index.html

```diff
@@ -2107,23 +2115,50 @@ <h2>Definition of Roles</h2>
+          For example, one common convention is that <kbd>Down Arrow</kbd> moves focus from the input to the first focusable descendant of the popup element.
+          If the popup element supports <pref>aria-activedescendant</pref>, in lieu of moving focus, such keyboard mechanisms can control the value of <pref>aria-activedescendant</pref> on the <code>combobox</code> element.
+          When a descendant of the popup element is active, authors MAY set <pref>aria-activedescendant</pref> on the <code>combobox</code> to a value that refers to the active element within the popup while focus remains on the <code>combobox</code> element.
+        </p>
 				<pre class="example highlight">
-				  &lt;div aria-label="Tag" role="combobox" aria-expanded="true" aria-owns="owned_listbox" aria-haspopup="listbox"&gt;
-				      &lt;input type="text" aria-autocomplete="list" aria-controls="owned_listbox" aria-activedescendant="selected_option"&gt;
-				  &lt;/div&gt;
-				  &lt;ul role="listbox" id="owned_listbox"&gt;
-				      &lt;li role="option"&gt;Zebra&lt;/li&gt;
-				      &lt;li role="option" id="selected_option"&gt;Zoom&lt;/li&gt;
+		      &lt;input type="text" aria-label="Tag"
+            role="combobox" aria-autocomplete="list"
+            aria-haspopup="listbox" aria-expanded="true"
+            aria-controls="popup_listbox" aria-activedescendant="selected_option"&gt;
+				  &lt;ul role="listbox" id="popup_listbox"&gt;
+			      &lt;li role="option"&gt;Zebra&lt;/li&gt;
+			      &lt;li role="option" id="selected_option"&gt;Zoom&lt;/li&gt;
 				  &lt;/ul&gt;
 				</pre>
-				<p>The ARIA 1.0 specification describes a <code>combobox</code> pattern where a text input element has the <code>combobox</code> role and owns a <rref>listbox</rref> element. <a>User agents</a>, <a>assistive technologies</a>, and conformance checkers SHOULD continue to support the ARIA 1.0 pattern so that existing implementations of the ARIA 1.0 pattern remain functional.</p>
+				<p>
+          The ARIA 1.0 specification describes a <code>combobox</code> pattern where the input element with the <code>combobox</code> role references the popup element with <pref>aria-owns</pref> instead of <pref>aria-controls</pref>.
+          <a>User agents</a>, <a>assistive technologies</a>, and conformance checkers SHOULD continue to support the ARIA 1.0 pattern so that existing implementations of the ARIA 1.0 pattern remain functional.
+          The ARIA 1.1 specification describes a <code>combobox</code> pattern where the element with role <code>combobox</code> is a composite container instead of a focusable input.
+          <a>User agents</a> and <a>assistive technologies</a> did not adequately support the ARIA 1.1 pattern, and this pattern supersedes the ARIA 1.1 pattern, which authors SHOULD abandon. 
```

Comment:
Not sure if "this pattern" is specific enough in the following phrase:
> and this pattern supersedes the ARIA 1.1 pattern

Consider changing to:
> so the ARIA 1.2 pattern supersedes the ARIA 1.1 pattern